setup.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine
+COPY . /home/src
+WORKDIR /home/src
+RUN pip3 install requests
```

Comment:
I think we should have a requirements.txt file here, so that you can do it with a virtualenv if so inclined, but admittedly that's just because I want an easy way to add YAML configuration back in. 😁

Reply:
added a card for this https://trello.com/c/fSty387c